tt_metal/hw/firmware/src/brisc.cc

```diff
@@ -343,6 +347,16 @@ int main() {
     int32_t num_words = ((uint)__ldm_data_end - (uint)__ldm_data_start) >> 2;
     l1_to_local_mem_copy((uint*)__ldm_data_start, (uint tt_l1_ptr*)MEM_BRISC_INIT_LOCAL_L1_BASE_SCRATCH, num_words);
 
+    int32_t num_dram_to_noc_words = (NUM_NOCS * NUM_DRAM_BANKS) << 1;
```

Comment:
let's put this code block in a fn in the same file as l1_to_local_mem_copy (since it gets reused).  noc_bank_table_init maybe.